app/frontend/errors.ts

```diff
@@ -0,0 +1,72 @@
+export enum Errors {
```

Comment:
In enum-like semantics, this should use singular (`Error`). Otherwise it is also strange when you use it like `getTranslation(code: Errors)` instead of `getTranslation(code: Error)`. As `code` implies singular and `Errors` plural it can become confusing.